src/test/java/Homework7/PractiseExpandingTestingTest.java

```diff
@@ -0,0 +1,52 @@
+package Homework7;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.openqa.selenium.interactions.Actions;
+
+import java.time.Duration;
+
+public class PractiseExpandingTestingTest extends  BasePracitceExpanding{
+    @Test
+    public void CheckIfTheAmountIsCorrect() throws InterruptedException {
```

Comment:
Please follow the naming conventions for method names. Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html